libexec/pyenv-versions

```diff
@@ -112,7 +112,7 @@ print_version() {
   if [[ -L "$path" ]]; then
     # Only resolve the link itself for printing, do not resolve further.
     # Doing otherwise would misinform the user of what the link contains.
-    version_repr="$version --> $(resolve_link "$path")"
```

Comment:
Using `resolve_link` here is intentional. As the comment says, `pyenv versions` should only display the link's contents, not the final path.

Reply:
I guess we can use `readlink` here -- since it's supposed to be always present and we don't need `-f` here.